corehq/apps/accounting/models.py

```diff
@@ -241,6 +241,7 @@ class BillingAccountAdmin(models.Model):
     domain = models.CharField(
         max_length=256, null=True, blank=True, db_index=True
     )
+    last_modified = models.DateTimeField(auto_now=True, default=datetime.datetime.utcnow())
```

Comment:
@czue 

Yes this is redundant. I added this, because I didn't want setup manually a default value in schemamigration. I updated models.